.github/workflows/check-generated-files.yml

```
@@ -0,0 +1,55 @@
+name: Check Auto Generated Files
+
+on:
+  push:
+    branches: [ 'main' ]
+  pull_request:
+    branches: [ '**' ]
```

Assim é suficiente :)

```suggestion
  pull_request
```